apis/collections/collections_resource.py

```
@@ -0,0 +1,194 @@
+    def delete(self, collection_name):
+        """
+        Endpoint that deletes a collection
+        """            
+        try:
+            current_app.logger.info('Deleting collection')
+            if (collection_name):
+                # collection_helper.delete_collection(collection_name)
+                return 'OK', 204
+            else:
+                return "Bad request, parameters missing", 400
+        except ValueError as e:
```

Ruff (0.13.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)